lib/scope.js

```diff
@@ -909,36 +921,73 @@ module.exports = {
+            + `value, but it's missing. The test might get unexpected results. See `
+            + `https://suffolklitlab.org/docassemble-AssemblyLine-documentation`
+            + `/docs/automated_integrated_testing/#trigger.`
+            + `\nThe row's data is\n${ JSON.stringify( match.original )}.`
+            + `\nThis field's data is\n${ JSON.stringify( field )}`;
+          await scope.addToReport(scope, { type: `warning`, value: row_trigger_warning });
+          fallbacks.push( match );
+        } else {
           let triggers_match = await scope.matchNeutralizedQuotes( scope, { value1: field.trigger, value2: match.trigger });
           if ( triggers_match ) {
             matches.push( match );
           }
         }
+      }  // ends for each match
+
+      // Be very permissive, based on feedback. See #464 comment.
+      // Use undefined trigger row(s) if no matching triggers were found
+      if ( matches.length === 0 ) { matches = fallbacks; }
+
+      // Feedback for devs on multiple row match without any matching triggers
+      if ( original_matches.length > 1 && matches.length === 0 ) {
+        let missing_row_warning = `The story table has multiple rows `
+          + `that match a specific field on the page with the id `
+          + `"${ scope.page_id }", but none match the trigger variable `
+          + `for the variable being set. Is the story table missing a row? `
+          + `The test will have to ignore this field. See `
+          + `https://suffolklitlab.org/docassemble-AssemblyLine-documentation`
+          + `/docs/automated_integrated_testing/#trigger.`
+          + `\nThe data of all the rows is\n${ JSON.stringify( original_matches )}.`
+          + `\nThis field's data is\n${ JSON.stringify( field )}`;
+        await scope.addToReport(scope, { type: `warning`, value: missing_row_warning });
       }
-      // Feedback for devs - we are missing the matching triggers we need
-      if ( matches.length === 0 ) { await scope.addToReport(scope, { type: `warning`, value: `A field on this page uses an index variable or a generic object. The test gave multiple matches for the story data for the field, but none match the trigger variable. Is the story table missing a row? For now, the test will have to ignore this field. The rows' data is\n${ JSON.stringify( original_matches )}.\nThe field's data is\n${ JSON.stringify( field )}` }); }
-    }
+    }  // ends if need trigger var
 
-    // Dev may have accidentally included a row more than once. Give them feedback
+    // Dev may have accidentally included a row more than once.
     if ( matches.length > 1 ) {
-      await scope.addToReport(scope, { type: `warning`, value: `there are multiple story data matches for this field. The story table data that appears last will be used. The field's data is\n${JSON.stringify( field )}` });
+      let multiple_matches_msg = `Multiple story table rows match this field's `
```

Comment:
We have decided to add "Do you have a duplicate story table row?" to the front of this error message.